core/py/src/braintrust_core/version.py

```diff
@@ -1 +1 @@
-VERSION = "0.0.26"
+VERSION = "0.0.27"
```

Comment:
this PR doesn't modify core. Is this unrelated or should it be bumping the SDK version?

Reply:
i did this because to give people a preview, I had to bump and publish core, so I figured i'd bundle into this change. It's likely cleaner to undo it here, and push a separate commit on main.